nss-cohort-40/banagzon-workforce-mgt-upper-class-twits
# Adi comp dets

# Description
Created the computer details page. HREFs from list link to the details
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Check into my branch and go to the computers tab on the nav bar, click on a computer to see the details
# Checklist:
